cmd/common.go

```diff
@@ -26,6 +26,19 @@ var (
 	assumeYes  bool
 )
 
+func performValidate(ctx context.Context, kongClient *kong.Client, entity string) (bool, error) {
+	endpoint := fmt.Sprintf("/schemas/%s/validate", entity)
+	req, err := kongClient.NewRequest("POST", endpoint, nil, nil)
```

Comment:
So you are making the right API call here but instead, you need to pass the entity as body.
I recommend playing with the endpoint directly in Kong to get a better feel of the API in Kong: https://docs.konghq.com/enterprise/2.5.x/admin-api/#validate-a-configuration-against-a-schema

And then write more code here to use it correctly.